.github/bin/github-releases-setup.sh

```diff
@@ -19,6 +19,7 @@
 
 set -ex
 
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
wouldn't just `SCRIPT_DIR=$(dirname $0)` work ?

Reply:
It wouldn't as `$0` would not contain the absolute path. Let's consider the following script:
```bash
#!/usr/bin/env bash                                                            
                                                                               
SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
echo "SCRIPT_DIR: $SCRIPT_DIR"                                                 
echo "Dollar zero: $0"                                                         
echo "Dirname dollar zero: $(dirname $0)"                                      
```
```shell
 ~/other/verible-bin .....................
> ./a/1.sh
SCRIPT_DIR: /home/adam/other/verible-bin/a
Dollar zero: ./a/1.sh
Dirname dollar zero: ./a
Hi there

```

Reply:
If you need the absolute path, you can still do `$(realpath $(dirname $0))`

But why do we need the absolute path ? You are not changing the directory.

Reply:
Now that I thought about it, we don't really need the absolute path as just like you said we're not changing the current directory anywhere. I have just replaced that bit with `SCRIPT_DIR=$(dirname $0)`.